docs/changelog.txt

```diff
@@ -1,7 +1,41 @@
 [[changelog]]
 [appendix]
-== Release Notes
+Release Notes
```

Comment:
Sorry I didn't catch this. Either way if you don't mind fixing it on 0.1 then I think we can merge it in with this PR or after. Otherwise I can take a stab at fixing (is it just changelog.txt?).